Bottom: 0fb59be3385f6a2ac69233665c7c090f2b076df2
Top:    9996ebe78d7f4a9ebc14825418c1ae670b27522d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Ignore exim4.conf.template

We don't use this file, so ignore it


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index 7080bd4..84e1ca0 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -21,7 +21,6 @@ chown -R root:mail /var/mail
 case ${1} in
     exim4)
         if   [[ -f ${CONFIG_PATH}/update-exim4.conf.conf ]] \
-          && [[ -f ${CONFIG_PATH}/exim4.conf.template ]] \
           && [[ ! -f ${CONFIG_PATH}/exim4.conf ]]
         then
             # set the hostname
